### PR TITLE
CVE-2015-1281

### PR DESCRIPTION
**Pull request by Moremothansasuke**
This file was mostly done before it was assigned to me but curated was not set to true.

### CODE REVIEW

File: cves/CVE-2015-1281.yml [hunk 1]
```diff
@@ -58,13 +58,13 @@ fixes_vcc_instructions: |
...
+  :note: Keeps track of isolates so that everything can be isolated properly.
 - :commit: 9fb1e9fc1f3829f9597f830e599fc40fadc84db3
-  :note: Reimplement fix this time it passed tests.  Last commit for this fix.
+  :note: Reimplement fix for properly setting ScriptState for image loading.
 - :commit: 7faa3067e4b664eb19a7a682697f5b387a130728
-  :note: 
+  :note: Properly set ScriptState for image loading.
 - :commit: 0f291b9f331a04c55e0d050bfaef20b1c6a6af1c
-  :note: Revert attempt at fix as it was failing tests.
+  :note: Revert 3rd commit in this list (7faa) due to possibly causing leaking issues.
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1281.yml]: Make sure notes are the commit messages

File: cves/CVE-2015-1281.yml [hunk 2]
```diff
@@ -58,13 +58,13 @@ fixes_vcc_instructions: |
...
 vccs:
 - :commit: 52aa3b1bcf3079fc72a2b24a73dff78017eb5e7a
   :note: Bug fix for issue 200923002.  The issue was fixed by adding a microtask to
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1281.yml]: Same as above, note should be the commit message

File: cves/CVE-2015-1281.yml [hunk 3]
```diff
@@ -110,17 +110,17 @@ discovered:
...
 subsystem:
   question: |
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
+  answer: Blink->SVG
   name: Blink->SVG
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1281.yml]: Change to "Blink, SVG"

File: cves/CVE-2015-1281.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1281.yml]: Remove timestamp from "announced" below (leave date)

Comment (CatherineOsadciw) [reply cves/CVE-2015-1281.yml]: and bounty date below

File: cves/CVE-2015-1281.yml [hunk 5]
```diff
@@ -58,13 +58,13 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: dff368031150a1033a1a3c913f8857679a0279be
-  :note: 
+  :note: Keeps track of isolates so that everything can be isolated properly.
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1281.yml]: Keeps track of isolates? Not sure what you're referring to here.

File: cves/CVE-2015-1281.yml [hunk 6]
```diff
@@ -171,36 +171,36 @@ lessons:
...
-    applies: 
-    note: 
+    applies:
+    note:
   yagni:
-    applies: 
-    note: 
+    applies:
+    note:
   complex_inputs:
```
Comment (kzalbak) [inline cves/CVE-2015-1281.yml]: Won't let me comment below, but I would consider talking about the mitigation further in your analysis. Talk about what the fix actually accomplished. It also seems really choppy with all the short sentences, consider expanding the sentences for better flow.

File: cves/CVE-2015-1281.yml [hunk 7]
```diff
@@ -110,17 +110,17 @@ discovered:
     leave the entries blank except for "answer". Write down where you looked in "answer".
   answer: Reported by someone not in Google partial email address of masatoki...@gmail.com.
```
Comment (bnk5096) [inline cves/CVE-2015-1281.yml]: Maybe refine this answer a little bit.
Perhaps: "Reported by someone not working at Google. The reporter has a partial email address of  ..."

File: cves/CVE-2015-1281.yml [hunk 8]
```diff
@@ -58,13 +58,13 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
```
Comment (bnk5096) [inline cves/CVE-2015-1281.yml]: I can't comment above, but maybe refine the description. it seems to be a bit high level. How is this dangerous javascript injected? 

If you don't decide to do a refactor here, `user's` should be either `a user's` or `users'`

File: cves/CVE-2015-1281.yml (hunk 5 above)
Comment (Moremothansasuke) [reply cves/CVE-2015-1281.yml]: An isolate is a chromium specific term that refers to a bunch of data that should be kept isolated and not confused with other data, like how each tab is separate from each other tab. I don't know if chromium specific terminology should be avoided here (or if it even can be), but for all isolates is a confusing name I think this is the best way to phrase this concisely. If that terminology should be avoided then I can describe what an isolate is here, though that feels a little out of scope for this section. If you think I should explain it please let me know and I can fix it.

File: cves/CVE-2015-1281.yml (hunk 6 above)
Comment (Moremothansasuke) [reply cves/CVE-2015-1281.yml]: Sorry, I am kind of confused about what part you are talking about. If you mean the mistakes section I edited how the original author had worded things, but I also agree that this is the most important thing to note in the mistakes section. I don't feel like the mistakes section is where analysis of what the fix actually accomplished should go, if you think it should be added where do you think it should go?

**Pull request merged**